### PR TITLE
Orgnize cluster parameters and add cluster_info attribute

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

File: openshift_cli_installer/libs/clusters/aws_ipi_cluster.py [hunk 1]
```diff
@@ -80,33 +80,64 @@ def _create_install_config_file(self):
             docker_config_file=self.docker_config_file,
         )
         self.ssh_key = get_local_ssh_key(ssh_key_file=self.ssh_key_file)
+
+        terraform_parameters = {
+            "name": self.cluster["name"],
```
Comment (myakove) [inline openshift_cli_installer/libs/clusters/aws_ipi_cluster.py:85]: We want to take all from `cluster_info`, no?

**Review by omrirh — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Pull request merged**